client-sdk/ts-web/rt/playground/src/index.js

```diff
@@ -98,7 +98,7 @@ const keyvalueWrapper = new Wrapper(KEYVALUE_RUNTIME_ID);
             const THE_KEY = oasis.misc.fromString('greeting-js');
             const THE_VALUE = oasis.misc.fromString('Hi from JavaScript');
 
-            const alice = oasis.signature.EllipticSigner.fromSecret(await oasis.hash.hash(oasis.misc.fromString('oasis-runtime-sdk/test-keys: alice')), 'this key is not important');
+            const alice = oasis.signature.NaclSigner.fromSeed(await oasis.hash.hash(oasis.misc.fromString('oasis-runtime-sdk/test-keys: alice')), 'this key is not important');
```

Comment:
@pro-wh This is imported from `core` - not sure if that breaks anything as I haven't played with `rt` yet

Reply:
seems like it should result in the same key, from what we discussed earlier. maybe we could add a check against a reference public key